.github/workflows/release.yml

```diff
@@ -72,10 +72,7 @@ jobs:
 
       - name: Install the latest AWS CLI
         if: ${{ startsWith(github.ref, 'refs/tags/') || github.ref == 'refs/heads/main' }}
-        run: |
-          apk add --no-cache python3 py3-pip
-          pip3 install --upgrade pip
-          pip3 install awscli
+        uses: unfor19/install-aws-cli-action@v1
```

Comment:
github comes with aws-cli preinstalled by default, we can fully drop this step IMO

https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#cli-tools

Reply:
ah they used to ship it with v1.0. I didn't know it's 2.0 now.